Update implementations.md

productml-ai opened the pull request:
Adding Azure Monitor in alphabetical order

productml-ai committed:
Update implementations.md

Adding Azure Monitor in alphabetical order

SergeyKanzhelev commented on implementations.md:
```diff
@@ -2,6 +2,17 @@
+**Website:** [Azure Monitor](https://azure.microsoft.com/en-us/services/monitor/)
+ 
+**Implementations:**
+.NET,
```
would you mind adding links to keep it consistent with the rest of the document?

productml-ai replied:
Yes. Waiting for confirmation on specification links before applying.

productml-ai replied:
Updated with available links

SergeyKanzhelev replied:
Do you want to wait for more links or PR is ready to be merged?

(we will need more approvals anyway, just asking to understand the status)

productml-ai replied:
Yes. I am in discussion with devs and we will probably move to "All technologies supported by Azure Monitor..." and link back to docs.microsoft.com since that is our source of truth for documentation. I should be able to close out by tomorrow.

productml-ai replied:
We decided that it is best to navigate the user to docs.microsoft.com

SergeyKanzhelev commented on implementations.md:
```diff
@@ -2,6 +2,17 @@
 This page contains an alphabetical list of projects and vendors that currently implement Trace Context.
 
+## Azure Monitor
+**Website:** [Azure Monitor](https://azure.microsoft.com/en-us/services/monitor/)
```
nit:

```suggestion
**Website:** [Azure Monitor](https://azure.microsoft.com/services/monitor/)
```

productml-ai committed:
Update implementations.md

Co-authored-by: Sergey Kanzhelev <S.Kanzhelev@live.com>

productml-ai committed:
Update implementations.md

add available links to supported languages

productml-ai committed:
Update implementations.md

Removed individual links and updated to navigate to Azure Monitor documentation support
